lab5/src/main/java/SearchArray.java

```diff
@@ -0,0 +1,149 @@
+/**
+ *	SearchArray implements a linear search in an array, searching
```

Comment:
Looks like you started lab7 on top of the lab 5 branch. I would just get rid of this file in the lab7 branch. Else it's all good.